pkg/rtc/participant_signal.go

```diff
@@ -178,6 +178,7 @@ func (p *ParticipantImpl) sendIceCandidate(c *webrtc.ICECandidate, target liveki
 	// write candidate
 	p.params.Logger.Debugw("sending ice candidates",
 		"candidate", c.String(), "target", target)
+	ci.Candidate += fmt.Sprintf(" network-cost %d", p.networkCost)
```

Comment:
Does this work on both Chrome and Safari? Wondering if instead hacking the priority directly would work better? But knowing the priority will be harder.

Reply:
yeah, hacking candidate is hard that we should make sure server side's priority less than client's (min priority takes high bits in priority calc), and perform all priority calculate to make sure new node's candidate to be used, and  the client's peer reflex candidate's priority is hard to known too.